src/test/java/io/quarkus/search/app/SearchServiceTest.java

```diff
@@ -118,41 +118,52 @@ public void relevance(String query, List<String> expectedGuideIds) {
 
     private static List<Arguments> relevance_params() {
         return List.of(
+                // I wonder if we could use something similar to https://stackoverflow.com/a/74737474/5043585
+                // to have some sort of weight in the documents and prioritize some of them
```

Comment:
I think you want https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-rank-feature-query.html
We need a PageRank :)

Reply:
Seriously though, we could implement a very simple rank based on which guide is referenced by others. The Hibernate ORM guide would automatically get a higher rank than e.g. Panache or Hibernate Search, and that's what we want.

Reply:
Created https://github.com/quarkusio/search.quarkus.io/issues/12